ajax/policyValue.php

```diff
@@ -42,9 +42,23 @@
 }
 
 $policyId = intval($_POST['value']);
+$mode = (isset($_POST['mode'])) ? strval($_POST['mode']) : null;
 
 $factory = new PluginFlyvemdmPolicyFactory();
 $policy = $factory->createFromDBByID($policyId);
 if ($policy !== null) {
```

Comment:
The content if this if() structure contains HTML output. Can you move all HTML generation code in a method of a class ?